File: uri/C/beginner/1046.c
#include <stdio.h>

int main()
{

	int inicio, fim, total;

	scanf("%d %d", &inicio, &fim);

	if(inicio >= 0 && inicio <= 24){
		if(fim >= 0 && fim <= 24){
			if(inicio >= fim){
				total = (24 + fim) - inicio;
				printf("O JOGO DUROU %d HORA(S)\n", total);
			}
			if(fim > inicio){
				total = fim - inicio;
				printf("O JOGO DUROU %d HORA(S)\n", total);	
			}
		}
	}

	return 0;
}

File: uri/python/1046.py
# -*- coding: utf-8 -*-

inicio, fim = input().split()

inicio = int(inicio)
fim = int(fim)

total = 0

if inicio >= 0 and inicio <= 24:
	if fim >= 0 and fim <= 24:
		if inicio >= fim:
			total = (24 + fim) - inicio
			print('O JOGO DUROU {} HORA(S)'.format(total))
		else:
			total = fim - inicio
			print('O JOGO DUROU {} HORA(S)'.format(total))

